### PR TITLE
fix(cypress): fix controller cypress tests

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Fix the cypress tests for controllers.

## QA

n/a

## Fixes

Fixes: #4036.

### CODE REVIEW

File: cypress/integration/with-users/controllers/list.spec.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ context("Controller listing", () => {
         cy.findByRole("link").click();
       });
 
-    cy.findByText(/Controller summary/).should("exist");
+    cy.findByText(/Summary/).should("exist");
```
Comment (huwshimi) [inline cypress/integration/with-users/controllers/list.spec.ts:31]: This label has changed for the React version.

File: cypress/integration/with-users/controllers/list.spec.ts [hunk 2]
```diff
@@ -77,7 +77,7 @@ context("Controller listing", () => {
         })
         .should("exist")
     );
-    [/Set zone/i].forEach((name) =>
+    [/Deploy/i].forEach((name) =>
```
Comment (huwshimi) [inline cypress/integration/with-users/controllers/list.spec.ts:80]: Setting a zone is valid for some controllers.

**Review by Caleb-Ellis — APPROVED** [review]

**Comment by huwshimi** [comment]
@petermakowski or @Caleb-Ellis do you have any ideas why this might not be finding the tag input when run in actions, but works locally?

**Comment by Caleb-Ellis** [comment]
@huwshimi the screenshots show that the placeholder is missing - since the Cypress tests are run using the edge snap (not this PR) then the change you've made here isn't reflected there yet.

**Comment by huwshimi** [comment]
> @huwshimi the screenshots show that the placeholder is missing - since the Cypress tests are run using the edge snap (not this PR) then the change you've made here isn't reflected there yet.

Oh, of course, that always gets me! Thanks a lot.
